### ISSUE TITLE
Incompatible with contemporary Angular/Node versions

### ISSUE THREAD

**Issue by ream720** [angular-in-action/datacenter#5]
On node 16 and Angular 14 this project won't `npm i` successfully. I know you said it is compatible with older versions, but I know you understand the frustration of juggling versions. Hopefully Manning would be willing to work with you to get resources updated because this book is so good! Potentially an Angular 14+ re-write/update? 

Thanks for all your work on this book. Even if I don't get the project running, it is still proving useful.

**Comment by gnomeontherun** [comment]
This is correct and known, the issue is these samples need to work with what is written in the book. Angular 15 is nearly out, and so it's been 5 years. I'm currently working on creating a new branch with necessary tweaks to work with Angular 15. So far, the only issues I've run into are tooling and TypeScript strictness, and the code from the book still works (if you fix the TypeScript issues). At a minimum, it will allow you to follow the new examples and run the code, and hopefully I can work that out soon (a bit loaded right now with grad school, work, family, etc 😀)

**Comment by gnomeontherun** [comment]
I'm stuck on this project because ng-bootstrap doesn't yet support Angular 15. Using dependencies was something I should have avoided doing in the book, because I can't control those things. I'll consider if I can do a more significant redo to avoid it or what may be a best path.
